exercises/pythagorean-triplet/src/test/java/PythagoreanTripletTest.java

```diff
@@ -20,6 +20,156 @@ public void shouldCalculateSum() {
         assertEquals(expected, actual);
     }
 
+    @Ignore("Remove to run test")
+    @Test
+    public void tripletsWhoseSumIs12() {
+        List<PythagoreanTriplet> actual
+                = PythagoreanTriplet
+                        .makeTripletsList()
+                        .withFactorsLessThanOrEqualTo(12)
+                        .thatSumTo(12)
+                        .build();
+
+        List<PythagoreanTriplet> expected
+                = Arrays.asList(
+                        new PythagoreanTriplet(3, 4, 5));
```

Comment:
This statement looks short enough to all be on one line 🙂 And you can use `Collections.singletonList` since the list just has one element 🙂